core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1607,6 +1708,22 @@ class TopicChangeListener(controller: KafkaController, eventManager: ControllerE
   }
 }
 
+/**
+  * Called when broker notifies controller of disk change
+  *
+  * @param controller
```

Comment:
This comment seems OK since it is similar to the comment of `BrokerChangeListener`.. Do you suggest to add comments with information similar to the comment of `PartitionReassignmentListener`?